src/main/java/org/symphonyoss/integration/webhook/jira/parser/v2/JiraMetadataParser.java

```diff
@@ -123,6 +182,25 @@ protected void preProcessInputData(JsonNode input) {
   }
 
   /**
+   * This method puts in the Jira's payload JSON the accent according to the issue type, defined in
+   * the accentMap initialization
+   * @param input
+   */
+  private void processAccent(JsonNode input) {
+    JsonNode fieldsNode = input.path(ISSUE_PATH).path(FIELDS_PATH);
+    if (fieldsNode.has(ISSUETYPE_PATH)) {
+      String issueName = fieldsNode.path(ISSUETYPE_PATH).path(NAME_PATH).asText();
+      if (StringUtils.isNotEmpty(issueName)) {
+        issueName = issueName.toLowerCase();
+
+        if (accentMap.containsKey(issueName)) {
+          ((ObjectNode) input).put(ACCENT_PATH, accentMap.get(issueName));
```

Comment:
You should include a default value to accent if you cannot determine the issue type.